tests/test_alternative_citations.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from nose.tools import *  # PEP8 asserts
+
+from website.project.model import AlternativeCitation
+
+from modularodm.exceptions import ValidationError
+
+from tests.base import OsfTestCase
+
+from tests.factories import UserFactory, ProjectFactory
+
+class ModelTests(OsfTestCase):
+
+    def setUp(self):
+        OsfTestCase.setUp(self)
+        self.user = UserFactory()
+        self.node = ProjectFactory(creator=self.user)
+        self.citation = AlternativeCitation(name='Initial Citation', text='This is my test citation')
+        self.citation.save()
+        self.node.alternativeCitations.append(self.citation)
+
+    def test_model_success(self):
+        alt_citation = AlternativeCitation(name='test', text='citation')
+        try:
+            alt_citation.save()
+            self.node.alternativeCitations.append(alt_citation)
+        except ValidationError:
```

Comment:
It should happen for the other ones, but not for test_model_success so I removed it from that but not the others.